src/sentry/static/sentry/app/views/settings/components/dataPrivacyRulesPanel/dataPrivacyRulesPanelForm/dataPrivacyRulesPanelFormSelectorFieldSuggestions.tsx

```diff
@@ -1,6 +1,14 @@
 import {t} from 'app/locale';
 
-const binaryOperatorSuggestions: Suggestions = [
+export type SuggestionType = 'value' | 'unary' | 'binary' | 'string';
+
+export type Suggestion = {
+  type: SuggestionType;
+  value: string;
+  description?: string;
+};
```

Comment:
Let's move these to a `types` module otherwise it will get difficult to know where to import types from